Makefile

```diff
@@ -87,6 +87,7 @@ deploy: manifests kustomize ## Deploy controller in the configured cluster
 	cd config/metallb_rbac && $(KUSTOMIZE) edit set namespace $(NAMESPACE)
 	$(KUSTOMIZE) build $(KUSTOMIZE_DEPLOY_DIR) | kubectl apply -f -
 	$(KUSTOMIZE) build config/metallb_rbac | kubectl apply -f -
+	hack/patch-openshift.sh $(NAMESPACE)
```

Comment:
@andreaskaris just a clarification. are we ok to roll out ocp specific changes after deploying the operator ? can we do with kustomize and then deploy ?

Reply:
Done :-)

Reply:
Thanks @andreaskaris. This is really nice! couple of nits.